src/manager/passwords.js

```diff
@@ -2,15 +2,29 @@ import React from 'react';
 import { decrypt } from './encryption';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faDownload, faTrash, faUpload } from '@fortawesome/free-solid-svg-icons';
+import { useState } from "react";
+import {CopyToClipboard} from 'react-copy-to-clipboard';
+import { faCopy } from '@fortawesome/free-solid-svg-icons'
 
 const Passwords = ({data, onUpdate, onImport, onExport, onDeleteAll}) => {
+    const [showPassModalData, setshowPassModalData] = useState("");
+    const [copied, setCopied] = useState(false);
+    const[showPassModal, setshowPassModal] = useState(false);
+    const[userPass, setUserPass] = useState("")
     const handleShow = (password) => {
         const key = prompt("Enter main password");
         if(key == null) {
             alert("Please enter password")
             return
         }
-        alert(decrypt(password, key));
+
+        setUserPass("" + decrypt(password, key))
```

Comment:
Why do we need to set it in state, we can directly pass the password to the modal